src/libtomahawk/audio/AudioOutput.cpp

```diff
@@ -37,7 +37,7 @@
 #include <vlc/libvlc_events.h>
 #include <vlc/libvlc_version.h>
 
-static const int ABOUT_TO_FINISH_TIME = 2000;
+static const float ABOUT_TO_FINISH_POS = 0.95f;
```

Comment:
Not exactly sure why you use that "abouttofinish" thing anyway. If anything, even if track ended abruptly I personally would prefer it to go to the next song anyway.
Also, what would be dangerous about it being set 3 minutes before end?
And with current 2000 threshold I often get it not set for gmusic tracks.